### PR TITLE
marketing-ify the post-install message

### PR DESCRIPTION
**Pull request by indirect**
/cc @segiddins @mrb

### CODE REVIEW

File: bundler.gemspec [hunk 1]
```diff
@@ -34,9 +34,9 @@ Gem::Specification.new do |s|
   s.require_paths = ["lib"]
 
   s.post_install_message = <<-END.lines.map(&:strip).join(" ")
-    Bundler and RubyGems.org are free for anyone to use, but maintaining them
-    costs more than $25,000 USD every month. Help us cover those costs so that
-    we can keep the gem ecosystem free for everyone:
-    https://ruby.to/support-bundler
+    Did you know that maintaining Bundler and RubyGems.org costs more than
```
Comment (segiddins) [inline bundler.gemspec]: Maintaining & improving, maybe?

Comment (indirect) [inline bundler.gemspec]: Yeah, that's good. 👍

**Comment by segiddins** [comment]
@mrb look good to you?

**Comment by mrb** [comment]
@segiddins Yep!

**Comment by segiddins** [comment]
@bundlerbot r+

**Pull request merged**